components/amount-buttons.jsx

```diff
@@ -53,7 +60,36 @@ var AmountOtherButton = React.createClass({
       this.props.onRadioChange();
     }
   },
+  onInputChange: function(e) {
+    var inputValue = e.currentTarget.value;
+    var amount = "";
+
+    if (/^(\d)*[\,]?(\d){0,2}$/.test(inputValue)) {
```

Comment:
It makes turning 1,00 -> 1.00 much harder. I need to start parsing and I kinda wanted to attempt a solution first that didn't parse.